dev-python/pandas/pandas-0.23.3.recipe

```diff
@@ -0,0 +1,139 @@
+SUMMARY="A powerful data analysis library for Python"
+DESCRIPTION="Pandas is a Python package providing fast, flexible \
+and expressive data structures designed to make working with \
+\"relational\" or \"labeled\" data both easy and intuitive. \
+It aims to be the fundamental high-level building block for \
+doing practical, real world data analysis in Python."
+HOMEPAGE="https://pandas.pydata.org/"
+COPYRIGHT="2008-2012 AQR Capital Management, LLC
+	Lambda Foundry, Inc.
+	PyData Development Team"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://files.pythonhosted.org/packages/3e/56/82c4d4c049294f87ebd05b65fdcbc9ed68bd23fb0a7e4469caf9a75d199f/pandas-0.23.3.tar.gz"
+CHECKSUM_SHA256="9cd3614b4e31a0889388ff1bd19ae857ad52658b33f776065793c293a29cf612"
+SOURCE_DIR="pandas-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	pandas$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	setuptools_python
+	setuptools_python3
+	cython_python
+	cython_python3
+	python_numpy
+	python3_numpy
+	dateutil_python
+	dateutil_python3
+	pytz_python
+	pytz_python3
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:gfortran$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+
+PYTHON_PACKAGES=()
+PYTHON_VERSIONS=()
+PYTHON_LIBSUFFIXES=()
+# We don't have python2 for secondaryArch
+if [ -z "$secondaryArchSuffix" ]; then
+	PYTHON_PACKAGES+=(python)
+	PYTHON_VERSIONS+=(2.7)
+	PYTHON_LIBSUFFIXES+=("")
+	BUILD_REQUIRES+="
+		setuptools_python
+		cython_python
+		python_numpy
+		dateutil_python
+		pytz_python
+		"
+	BUILD_PREREQUIRES+="
+		cmd:python2
+		"
+fi
+# gcc2 does not support the flags passed by python3
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+	PYTHON_PACKAGES+=(python3)
+	PYTHON_VERSIONS+=(3.6)
+	PYTHON_LIBSUFFIXES+=(m)
+	BUILD_REQUIRES+="
+		setuptools_python3
+		cython_python3
+		python3_numpy
+		dateutil_python3
+		pytz_python3
+		"
+	BUILD_PREREQUIRES+="
+		cmd:python3
+		"
+fi
+for i in "${!PYTHON_PACKAGES[@]}"; do
+	pythonPackage=${PYTHON_PACKAGES[i]}
+	pythonVersion=${PYTHON_VERSIONS[$i]}
+	pythonLibSuffix=${PYTHON_LIBSUFFIXES[$i]}
+
+	eval "PROVIDES_$pythonPackage=\"
```

Comment:
Suggestion:
```diff
 	BUILD_PREREQUIRES+="
 		cmd:python3
 		"
 fi
+if [ "$targetArchitecture" = x86_gcc2 -a -n "$secondaryArchSuffix" ]; then
+	PROVIDES_python3="
+		pandas_python3 = $portVersion
+		"
+fi
 for i in "${!PYTHON_PACKAGES[@]}"; do
 	pythonPackage=${PYTHON_PACKAGES[i]}
 	pythonVersion=${PYTHON_VERSIONS[$i]}
 	pythonLibSuffix=${PYTHON_LIBSUFFIXES[$i]}
 
-	eval "PROVIDES_$pythonPackage=\"
+	eval "PROVIDES_$pythonPackage+=\"
 		${portName}_$pythonPackage = $portVersion
 		\""
```
This is a neutral change. Its sole purpose is to make `package list -i pandas_x86_python3-0.23.3-1-x86_gcc2.hpkg` show the 2 provides (`pandas_python3` and `pandas_x86_python3`) in 2 consecutive lines,  instead of having `pandas_x86_python3` at the top and `pandas_python3` at the bottom.

Note: The 5 lines inserted in the above diff are almost the same as the 5 lines removed a bit later. The difference is the `=` instead of a `+=`.

I used that same "trick" in `dev-python/coverage`:
https://github.com/haikuports/haikuports/blob/7ab967fb8d719a236b976c8c25edaa28cf81a000/dev-python/coverage/coverage-4.5.2.recipe#L58..L68
(but had forgotten that I had not used it for `dev-python/cryptography`.)

Oops, I just realised that this recipe does not provide any `cmd:` item, so the only effect of this suggestion will be that it will swap the order of the two provides, for `pandas_x86_python3`, on x86_gcc2.
That said, it might still be worth applying this suggestion, in case this recipe is later used as a template for another Python module providing `cmd:` items and having support for x86 secondary arch.